attractions/text-field/text-field.scss

```diff
@@ -119,14 +119,13 @@
 
     label {
       font-family: vars.$font;
-      background-color: #fff;
```

Comment:
Oh, did we really have this hardcoded? Oops. We have a variable for the background already though, `$background`, maybe we could use that here? I can't imagine a usecase where this would be different from the main background of the page.